### PR TITLE
style(global): apply design

### PR DESCRIPTION
**Pull request by ZonaHex**
add header(nav-bar); change background color

### CODE REVIEW

**Comment by ZonaHex** [comment]
Having trouble to use native tree icon, so I delete native title and move everything into 'extra'.

File: src/views/dashboard/data-explorer/components/table-list.vue [hunk 1]
```diff
@@ -50,5 +57,40 @@ a-tree(v-if="!ifTableLoading" :key="tableKey" :data="tableList" :load-more="load
...
+    'PRIMARY KEY': '/src/assets/images/key-icon.png',
+    'TIME INDEX': '/src/assets/images/time-icon.png',
+  }
   initTableDataSet()
 </script>
+<style lang="less" scoped>
+  .data-type {
+    height: 32px;
+    font-size: 12px;
+    font-family: PingFangSC-Regular, PingFang SC;
```
Comment (sunng87) [inline src/views/dashboard/data-explorer/components/table-list.vue:72]: update font

File: src/store/modules/user/index.ts [hunk 2]
```diff
@@ -13,15 +12,13 @@ const useUserStore = defineStore('user', {
...
-
-    // Get user's information
-    async info() {},
+    // switchRoles() {},
+    // // Set user's information
+    // setInfo() {},
+    // // Reset user's information
+    // resetInfo() {},
+    // // Get user's information
+    // async info() {},
```
Comment (sunng87) [inline src/store/modules/user/index.ts:21]: consider removing this if no longer used

**Review by sunng87 — APPROVED** [review]

**Comment by ZonaHex** [comment]
Found one problem: copy icon at TABLE tree will have weird borders if clicked. Not sure what went wrong.

**Pull request merged**